source/indexes/single-field-index.txt

```diff
@@ -0,0 +1,96 @@
+.. _php-single-field-index:
+
+====================
+Single Field Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+Single field indexes are indexes with a reference to a single field of a
+document in a collection. These indexes improve single field query and
+sort performance, and support :manual:`TTL Indexes </core/index-ttl>`
+that automatically remove documents from a collection after a certain
+amount of time or at a specified clock time.
```

Comment:
S: I think this might work better broken into two sentences since it's a pretty long sentence as-is
```suggestion
document in a collection. These indexes improve single field query and
sort performance. They also support :manual:`TTL Indexes </core/index-ttl>`
that automatically remove documents from a collection after a certain
amount of time or at a specified clock time.
```